### PR TITLE
Fix call to undefined function json_last_error_message

### PR DESCRIPTION
**Pull request by dawehner**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by xabbuh** [comment]
:+1: 

Status: Reviewed

File: src/Symfony/Component/Serializer/Tests/Encoder/JsonDecodeTest.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+use Symfony\Component\Serializer\Encoder\JsonDecode;
+
+class JsonDecodeTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var \Symfony\Component\Serializer\Encoder\JsonDecode */
+    private $decoder;
+
+    protected function setUp()
+    {
+        $this->decoder = new JsonDecode(TRUE);
```
Comment (xabbuh) [inline src/Symfony/Component/Serializer/Tests/Encoder/JsonDecodeTest.php]: `true` (lowercased)

File: src/Symfony/Component/Serializer/Tests/Encoder/JsonDecodeTest.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            'hello' => 'world',
+        ));
+        $result = $this->decoder->decode($json, 'json');
+        $this->assertEquals(array(
+            'hello' => 'world',
+        ), $result);
+    }
+
+    /**
+     * @expectedException \UnexpectedValueException
```
Comment (stof) [inline src/Symfony/Component/Serializer/Tests/Encoder/JsonDecodeTest.php]: this should be ``Symfony\Component\Serializer\Exception\UnexpectedValueException``

**Comment by dawehner** [comment]
Thank you for the review! Fixed those points, hopefully

**Comment by fabpot** [comment]
Thank you @dawehner.

**Comment by dawehner** [comment]
Thank you, that was quick!

On Wed, Nov 18, 2015 at 11:06 AM, Fabien Potencier <notifications@github.com

**Pull request closed without merging**